### PR TITLE
Latest fixes and updates for v3.0.1

### PR DESCRIPTION
**Pull request by jason-moore-hs**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,21 +1,18 @@
...
-organization := "com.hootsuite"
-
-scalaVersion := "2.10.3"
+crossScalaVersions := Seq("2.10.5", "2.11.6")
 
-publishTo := Some(Resolver.file("HootSuite repo", file("/var/www/maven-development")))
+organization := "com.hootsuite"
 
-resolvers += "Typesafe Repo" at "http://repo.typesafe.com/typesafe/releases"
+publishTo := Some("HootSuite Local repo" at "http://maven.hootops.com:8081/artifactory/hootsuite-local")
```
Comment (jason-moore-hs) [inline build.sbt]: Should we expose this URL?  Previously this project was publishing to a local maven repo only.

**Comment by eddsteel** [comment]
:+1:

File: build.sbt [hunk 2]
```diff
@@ -1,21 +1,18 @@
...
 
-scalaVersion := "2.10.3"
+organization := "com.hootsuite"
 
 publishTo := Some(Resolver.file("HootSuite repo", file("/var/www/maven-development")))
 
-resolvers += "Typesafe Repo" at "http://repo.typesafe.com/typesafe/releases"
-
-resolvers += "Sonatype" at "https://oss.sonatype.org/content/repositories/releases/"
+credentials += Credentials(Path.userHome / ".ivy2" / ".credentials")
```
Comment (hootsuitemike) [inline build.sbt]: Oh, awesome, you took these out and squashed the commit so it won't show up in the (public) OSS release git history.

File: src/main/resources/reference.conf [hunk 3]
```diff
@@ -1,6 +1,6 @@
 statsd {
-  host = "graph.hootsuitemedia.com"
-  port = 8125
+  # host = "graph.your.org"
+  # port = 2003
```
Comment (hootsuitemike) [inline src/main/resources/reference.conf]: Nice.

**Pull request merged**